src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import './Header.scss';
+import React from 'react';
+import '../../Functions/Sticky';
+
+function Header() {
+    return (
+        <div className="header">
+            <div className="header-logo">
```

Comment:
I guess according to BEM rules, this class should be named 'header__logo' and in scss: instead of .header-logo, we should put it inside .header class and name it &__logo.

Reply:
Ruta, you're right, I have to fix it a bit. But there are two possible ways. One of them you've just mentioned. But I like another one block--modifier-value. This one is more clear for me. Maybe it would be better to discuss with all team which one we will use. What do you think?